test.js

```diff
@@ -1,2 +1,9 @@
 console.log('test.js')
-coconuy()
\ No newline at end of file
+coconuy()
+
+var x = 10
+var y = 5
+if x > y 
+  console.log("x is greater than y")
```

Comment:
:robot: OpenAI

There is a missing semicolon at the end of line 7. It should be `if (x > y) {`.

<!-- This is an auto-generated comment by OpenAI -->